contracts/examples/Create2OwnableFactory.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity ^0.5.0;
+
+import "../utils/Create2.sol";
+import "../ownership/Ownable.sol";
+
+contract Create2OwnableFactory {
+  function deploy(bytes32 salt, bytes memory code) public {
```

Comment:
If I use `uint256` rather than `bytes32`, is it OK?

Reply:
@Skyge I believe it will be fairly common to calculate the salt with a `keccak256` hash. Is there a case where the salt would be calculated as an integer?